Jellyfin.Api/Helpers/MediaInfoHelper.cs

```diff
@@ -385,6 +385,21 @@ public void SortMediaSources(PlaybackInfoResponse result, long? maxBitrate)
     /// <returns>A <see cref="Task"/> containing the <see cref="LiveStreamResponse"/>.</returns>
     public async Task<LiveStreamResponse> OpenMediaSource(HttpContext httpContext, LiveStreamRequest request)
     {
+        // Enforce more restrictive transcoding profile for LiveTV due to compatability reasons
+        // Cap the MaxStreamingBitrate to 20Mbps, because we are unable to reliably probe source bitrate,
+        // which will cause the client to request extremely high bitrate that may fail the player/encoder
+        request.MaxStreamingBitrate = request.MaxStreamingBitrate > 20000000 ? 20000000 : request.MaxStreamingBitrate;
+
+        if (request.DeviceProfile is not null)
+        {
+            // Remove all fmp4 transcoding profiles, because it causes playback error and/or A/V sync issues
+            // Notably: Some channels won't play on FireFox and LG webOs
+            // Some channels from HDHomerun will experience A/V sync issues
+            var profileForCompatibility = request.DeviceProfile.TranscodingProfiles.ToList();
+            profileForCompatibility.RemoveAll(p => p.Container == "mp4");
+            request.DeviceProfile.TranscodingProfiles = profileForCompatibility!.ToArray();
```

Comment:
```suggestion
            request.DeviceProfile.TranscodingProfiles = request.DeviceProfile.TranscodingProfiles.Where(p => !string.Equals(p.Container, "mp4", StringComparison.OrdinalIgnoreCase)).ToArray();
```
Instead of creating list to just throw away